tests/stubtest_allowlists/py37.txt

```diff
@@ -159,3 +159,6 @@ typing_extensions\.reveal_type
 
 # Doesn't exist at runtime
 typing\.Protocol
+
+# Function at runtime; we pretend it's already a class
```

Comment:
This comment isn't accurate. `typing_extensions.NewType` _is_ a class on all versions of Python, as of v4.6.0:

```pycon
Python 3.8.16 (default, Jan 17 2023, 22:25:28) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from typing_extensions import NewType
>>> NewType
<class 'typing_extensions.NewType'>
```

Reply:
I see, the issue is that in the typing_extensions stubs we always just re-export typing.NewType, which is a function in the stubs below 3.10. I suppose I should change the stub instead.

Reply:
Yup, that's the conclusion I just came to as well.